### DISCUSSION TITLE
v1.0.3

### DISCUSSION THREAD

**Discussion by Kray-G** [Kray-G/kinx#312] [Archive]
This is a minor update to fix some bugs or improvements.

*   Improvements
    *   #308: Improvement of the stack usage.
    *   Some feedbacks from V1.1.0.
        *   Supported putting a comma at the end of an argument's list for both declaration and calling.
*   Bug Fixed
    *   #302: Fixed a prblem of the string optimization.
    *   #305: Fixed a prblem of no POPC in try.

<hr /><em>This discussion was created from the release <a href='https://github.com/Kray-G/kinx/releases/tag/v1.0.3'>v1.0.3</a>.</em>

**Comment by htsign** [comment]
既に2週間も前の出来事なので今更ですが、 `.github/workflows/scoop-bucket-update.yml` の実行に失敗してしまっていますね。
おそらくコミットハッシュを計算して `git push` する前に何らかのコミットが先に入ってしまったのだと思いますが、それを抜きにしても v1.0.0 からバージョンが動いていないようです。
手元のフォークリポジトリ htsign/kinx でワークフローをキックすると[正しく v1.0.3 に上がった](https://github.com/htsign/kinx/commit/adfb919378b00bf13b18b337740665bc36d031ca)のでよく分かりません…。

**Reply by Kray-G** [reply]
コメントありがとうございます。

CIプロセスが走った後、リリースする前に私が別ブランチにコミットしてしまったからかな、と思ってました。:sweat_smile:
たぶんワークフロー自体に問題は無いと思っているので、次回ちょっと確認してみますね。

**Reply by htsign** [reply]
`commit to main` と `push` の間に `git pull --rebase` を挟む修正コミットくらいは入れてもいいかなと思いました。
時間があれば私の方でそのうちPR出そうかなと思います🙇‍♂️
